project/settings.py

```
@@ -1,13 +1,22 @@
 from pathlib import Path
 from decouple import config
-
 import dj_database_url
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)